### PR TITLE
Refactor: move capture page to /captures

### LINKED ISSUE #1768: Move capture page path from: `/v2/captures/{id}` to `/captures/{id}`
http://localhost:3000/v2/captures/0c2d6e3a-d02a-4edf-bcdc-893e1f7a4817?bounds=-122.51961708068849,38.3588710328682,-122.50416755676271,38.370950949611135

this is current capture page, we need move the path to `/captures/{id}`, also, please write cypress integration test to cover this page.

---
some hints:
- Please read this before you start: https://docs.greenstand.org/web-map/v2-web-map
- If you want to join the Slack community (some resources need the community member's permission), please leave your email address.

### PR DESCRIPTION
**Pull request by yunchipang**
# Description

Fixes #1768 

## Type of change

[comment]: # 'Please delete options that are not relevant.'

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Screenshots

Screenshot 2024-11-04 at 6 58 02 PM

# How Has This Been Tested?

- [x] Cypress integration
- [ ] Cypress component tests
- [ ] Jest unit tests

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/components/FeaturedTreesSlider/index.js [hunk 1]
```diff
@@ -80,7 +80,7 @@ function FeaturedTreesSlider({ trees, size = null, isMobile, link }) {
       >
         {trees.map((tree) => (
           <Link
-            href={link ? link(tree) : `/v2/captures/${tree.id}`}
+            href={link ? link(tree) : `/captures/${tree.id}`}
```
Comment (yunchipang) [inline src/components/FeaturedTreesSlider/index.js:83]: @dadiorchen this is what you're looking for?
cypress integration test is not added yet. just checking if i'm on the right path.

Comment (dadiorchen) [reply src/components/FeaturedTreesSlider/index.js:83]: @yunchipang yes, and to finish this, we also need a integration test for this: /capture page

Comment (dadiorchen) [reply src/components/FeaturedTreesSlider/index.js:83]: Can you add it?

**Review by dadiorchen — COMMENTED** [review]
@yunchipang this is the right way to go, I guess you passed it on your local env, but on the ci it's broken: 
image
https://github.com/Greenstand/treetracker-web-map-client/actions/runs/11695766183/job/32571757532?pr=1804#step:3:147

can you take a look

File: cypress/tests/integration/captures/[captureid].cy.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+import capture from '../../../fixtures/capture.json';
+import country from '../../../fixtures/country.json';
+import grower from '../../../fixtures/grower.json';
+import { prepareNocks, clearNocks } from '../nockRoutes';
+
+beforeEach(() => {
+  clearNocks();
+});
+
+describe('Capture page', () => {
```
Comment (yunchipang) [inline cypress/tests/integration/captures/[captureid].cy.js:11]: @dadiorchen cypress integration test for the capture page has been added and passed. could you kindly take a look? i have requested another review from you. thanks!

**Pull request merged**